ats/src/atsMachines/slurmProcessorScheduled.py

```diff
@@ -535,7 +561,7 @@ def calculateCommandList(self, test):
                 #my_nodes="--nodes=%i" % (num_nodes) SAD TESTING
                 my_nodes="--epilog=none"
                 if (self.strict_nn == False):
-                    if self.slurmSeesATSProcessAsUsingACore == True:
+                    #if self.slurmSeesATSProcessAsUsingACore == True:
                         my_nodes="--nodes=%i-%i" % (num_nodes, self.numNodes)
```

Comment:
Commenting out this if statement leaves the "my_nodes= ..." line over indented. I think this will still run (even in Python 3) but it's confusing to the reader if this comment was intentional.

Reply:
I'll fix this.

Reply:
Fixed, rebased, and pushed